cmd/perunnodecli/README.md

```diff
@@ -0,0 +1,280 @@
+   perunnode-cli:
+
+```
+git clone https://github.com/hyperledger-labs/perun-node.git
+
+cd perun-node
+
+make
+```
+
+3. Generate configuration artifacts for demo artifacts for node and session
+   configuration:
+
+```
+./perunnode generate
+```
+
+This will generate the following artifacts:
+
+- Node: node.yaml file.
+- Session: Two directories (alice and bob) each containing session.yaml file,
+  contacts.yaml file and keystore directory with keys corresponding to the
+  on-chain and off-chain accounts.
+
+4. Start the perunnode:
+
+```
+./perunnode generate
+```
+
+This will start the perunnode using the config file located at default path
+`./node.yaml` that was generated in step 3. You will see a line
+"Serving payment channel API via grpc at port :50001" with a blinking cursor.
+Leave this running in this terminal.
+
+Now all the pre-requisites for `perunnode-cli` are setup.
+
+### Opening a session, opening channel within it, making payments & closing it
+
+1. Open two new terminals side by side, once each for alice and bob roles
+   respectively. In both the terminal, start the perunnode-cli app using below
+command:
+
+```
+./perunnodecli
+```
+
+This will bring up an interactive shell with auto-completion support. Type
+`help` to see a list of commands and their help message. Typing one of those
+commands without any arguments will print the help message for that command,
+including the list of sub-commands.
+
+2. In any one of the terminals, deploy perun contracts on the ganache-cli node
+   using the below commands. Just a reminder for one last time, you can almost
+   get every value by using auto-completion (by pressing TAB) and get away
+   without typing.
+
+```
+chain deploy-perun-contracts ws://127.0.0.1:8545
+```
+
+From here on, choose one terminal for alice role and one for bob role. In each
+step, the role will be the enclosed in square brackets before description.
+
+3. Opening a session and reading contact.
+
+- (a) [Alice] Start the session and get the contact of bob to check if it is
+   present. Getting the contact will also add the peer alias to auto-completion
+   list. The alias will then suggested, wherever a peer alias is expected. Two
+   exceptions where peer alias is not auto-completed are `contact add` and
+   `contact get` commands, because these commands are designed to add/get
+    contacts for unknown aliases.
+
+```
+# [Alice]
+node connect :50001
+session open alice/session.yaml
+contact get bob
+```
+
+- (b) [Bob] Repeat step 3 for bob using below commands:
+
+
+```
+# [Bob]
+node connect :50001
+session open bob/session.yaml
+contact get alice
+```
+
+5. Sending a request to open a payment channel and accepting it.
+
+- (a) [Alice] Send a request to open a channel with bob:
+
+```
+# [Alice]
+channel send-opening-request bob 1 2
+```
+
+- (b) [Bob] Receives a channel opening request notification that includes
+  request ID. Accept it:
```

Comment:
```suggestion
  request ID. Type the command to accept the channel opening request directly after receiving the notification:
```

Reply:
Signed-off-by: Manoranjith <ponraj.manoranjitha@in.bosch.com>